### PR TITLE
feature(toSlpRegtestAddress):

### PR DESCRIPTION
**Pull request by SuperCipher**
exposing toSlpRegtestAddress from bchaddrjs-slp

### CODE REVIEW

File: lib/utils.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import * as Bchaddr from "bchaddrjs-slp";
+import * as Bchaddr from "bchaddrjs-slp-2";
```
Comment (jcramer) [inline lib/utils.ts:1]: Why is this library being used? Afaict there isn't any difference other than the -2 version is behind by 1 commit.

File: tsconfig.json [hunk 2]
```diff
@@ -6,15 +6,16 @@
     "downlevelIteration": true,
     "moduleResolution": "node",
     "esModuleInterop": true,
-    "lib":[ "es2017" ],
+    "lib":[ "es2017","dom" ],
```
Comment (jcramer) [inline tsconfig.json:9]: why is "dom" being included?

File: tsconfig.json [hunk 3]
```diff
@@ -6,15 +6,16 @@
...
     "declaration": true,
     "sourceMap": true,
     "plugins": [
       { "name": "typescript-tslint-plugin" }
     ]
   },
   "include": [
-    "lib/**/*.ts"
+    "lib/**/*.ts",
+    "./typings-custom/**/*.ts"
```
Comment (jcramer) [inline tsconfig.json:18]: I'm not able to find any "typings-custom" files.  Why is this needed?

Comment (SuperCipher) [reply tsconfig.json:18]: Sorry, I forgot to remove that, it's actually not necessary.

**Comment by jcramer** [comment]
Closing this, as we've discussed elsewhere. See https://github.com/simpleledger/SLPDB/issues/82.
